### PR TITLE
test: change port to random

### PR DESCRIPTION
**Pull request by xtx1130**
ref: https://github.com/fastify/fastify/pull/3737#issuecomment-1056934583

#### Checklist

- [x] run `npm run test` and `npm run benchmark`
- [x] tests and/or benchmarks are included
- [x] documentation is changed or added
- [x] commit message and code follows the [Developer's Certification of Origin](https://github.com/fastify/.github/blob/master/CONTRIBUTING.md#developers-certificate-of-origin-11)
      and the [Code of conduct](https://github.com/fastify/.github/blob/master/CODE_OF_CONDUCT.md)

### CODE REVIEW

**Comment by mcollina** [comment]
Can you try rebase?

**Comment by xtx1130** [comment]
> Can you try rebase?

like this?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ on:
     paths-ignore:
       - 'docs/**'
       - '*.md'
-  pull_request_target:
+  pull_request:
```
Comment (jsumners) [inline .github/workflows/ci.yml:8]: Why is this file changed?

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -27,6 +27,7 @@ jobs:
 
       - name: Lint code
         run: |
+          echo 'this is an update'
```
Comment (jsumners) [inline .github/workflows/ci.yml:30]: What is this?

File: test/als.test.js [hunk 3]
```diff
@@ -31,7 +31,7 @@ app.post('/', function (request, reply) {
   reply.send({ id })
 })
 
-app.listen(3000, function (err, address) {
+app.listen(0, function (err, address) {
```
Comment (jsumners) [inline test/als.test.js:34]:
```suggestion
app.listen({ port: 0 }, function (err, address) {
```

**Review by jsumners — CHANGES_REQUESTED** [review]

File: .github/workflows/ci.yml (hunk 2 above)
Comment (xtx1130) [reply .github/workflows/ci.yml]: A test for github ci update. I have seen Mcollina has new commit to delete this, and I will have a merge

File: .github/workflows/ci.yml (hunk 1 above)
Comment (xtx1130) [reply .github/workflows/ci.yml]: ref: https://github.com/fastify/fastify/pull/3741. And I rebased from master

**Review by jsumners — APPROVED** [review]

**Review by Eomm — APPROVED** [review]
